### PR TITLE
PE-15 OJ: Verify for options presence by name (text)

### PR DESCRIPTION
**Pull request by danciu**

### CODE REVIEW

File: spec-data/specs2.json [hunk 1]
```diff
@@ -40,6 +40,17 @@
       "testFor": [{
         "clickable": "linkText=I’m new to BT"
       }]
+    },
+    {
+      "name": "OJ option names test",
+      "globalUrl": "https://www.productsandservices.bt.com/consumerOrders/control/orderproduct?productId=CON-27455&productKey=CON-DUAL&BD_BP=S0340394&BD_SOS=UWCUINF1_25_12M10L_A,UWCBB_13_12M450L,UWCUBB_18_12M5L,UEWCUBBX_26_12M16L&genericKey=CON-GEN-DUAL-FTTC&wantsInfinity=Y&altJourney=INFBUNDLE&showfibre=Y&showFTTP=Y&showEmail=Y&BT_SPORT_CUST_DETAILS_SWITCH=ON&SelfInstall=Trial&MJ=OJ2&EC=N&AMC=N&addNetflix=N&SS=N&ADDON=N&DEV=N&INT=N&LR_OPTION=SLR&CLD=N&DONGLE5=N",
```
Comment (danciu) [inline spec-data/specs2.json]: Link I manually got from the URL bar after clicking on a Presales package, so I suppose it should be OK to publicly have it here for dev testing. Am I wrong? Thanks

Comment (azz0r) [inline spec-data/specs2.json]: Should be fine as long as its not a Toby url ;)

Comment (danciu) [inline spec-data/specs2.json]: :+1:

File: spec-modules/oj-options-name.spec-module.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/**
+ * Click then wait spec module
```
Comment (danciu) [inline spec-modules/oj-options-name.spec-module.js]: Will have to change this

File: spec-modules/oj-options-name.spec-module.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    format.formatModuleDetail(format.getDetailFromArray(arrOptionNames)) + '...', function() {
+      var elemsUserChoice = element.all(by.deepCss('.formRow .userChoice'));
+
+      elemsUserChoice.each(function(elem, index) {
+        // Get the ID so we can associate it with a label - `input` elements are connected
+        // to their `label` element through the `id` and `for` HTML attributes
+        elem.getAttribute('id').then(function(strIdAttr) {
+          // Get the corresponding label for the current input
+          var elemCorrespondingLabel = element(by.xpath('//label[@for="' + strIdAttr + '"]'));
+
```
Comment (danciu) [inline spec-modules/oj-options-name.spec-module.js]: Also, just realised I also need to include the other type of selection element as well i.e. the one with tables, just like I have to do for the one I'm currently working on, `PE-18`. Easy change after finishing with that one too.

Comment (danciu) [inline spec-modules/oj-options-name.spec-module.js]: Also, just realised I also need to include the other type of selection element as well i.e. the one with tables, just like I have to do for the one I'm currently working on, `PE-18`. Easy change after finishing with that one too.

**Comment by danciu** [comment]
We can merge this now  @azz0r  :)

**Pull request merged**